ISICkonto/Flow/ViewModels/BalanceViewModel.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BalanceViewModel.swift
+//  ISICkonto
+//
+//  Created by Vendula Švastalová on 21/03/2019.
+//  Copyright © 2019 Vendula Švastalová. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import Kanna
+import SwiftKeychainWrapper
+import SVProgressHUD
+
+class BalanceViewModel: AppViewModel {
+    var refreshAction: Variable<Void> = Variable(())
+    
+    var disposeBag = DisposeBag()
+    
+    var balance : Observable<String> {
+        return resultRequestBalancePage.map { [weak self] in
+            guard let strongSelf = self, let balancePage = $0.element, $0.info == .loggedIn else {
+                SVProgressHUD.showError(withStatus: "Failed".localized)
+                SVProgressHUD.dismiss(withDelay: 1.0)
```

Comment:
Ten progress hud z VM není moc good. Když ti teď řeknu ať to vyměníš za nějaký custom view, tak jseš nahraná. Loading indicator je UI, tedy z těch písmenek MVVM je to to View, tak ho nedávej do VM 😏

Reply:
jasne, makes sense, ja prave vahala jak to s tim udelat :D fixnu